Userland/Utilities/cat.cpp

```diff
@@ -11,15 +11,36 @@
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 
+struct LineTracker {
+    size_t line_count = 1;
```

Comment:
fyi: `size_t line_count { 1 };` is what we use in most other places. I'll merge this, but if you end up touching this file again, maybe you could fix it up then.